debezium-connector-binlog/src/test/java/io/debezium/connector/binlog/util/BinlogTestConnection.java

```diff
@@ -15,14 +15,15 @@
 import io.debezium.connector.binlog.jdbc.BinlogConnectorConnection;
 import io.debezium.jdbc.JdbcConfiguration;
 import io.debezium.jdbc.JdbcConnection;
+import io.debezium.util.ThreadNameContext;
 
 /**
  * @author Chris Cranford
  */
 public abstract class BinlogTestConnection extends JdbcConnection {
 
     public BinlogTestConnection(JdbcConfiguration config, ConnectionFactory factory) {
-        super(addDefaultSettings(config), factory, "`", "`");
+        super(addDefaultSettings(config), factory, "`", "`", new ThreadNameContext("test-connector", "${debezium}-${connector.name}-${task.id}", "0"));
```

Comment:
Let's add the ${connector.name} and ${task.id} on top of the default thread name pattern. This way all the arguments would get tested while running the tests.